spyder/plugins/variableexplorer/widgets/collectionseditor.py

```diff
@@ -290,9 +295,12 @@ def canFetchMore(self, index=QModelIndex()):
         else:
             return False
 
-    def fetchMore(self, index=QModelIndex()):
+    def fetchMore(self, index=QModelIndex(), number_to_fetch=None):

```

Comment:
`fetchMore` is a Qt method, so I don't we should change its signature. So I don't agree with this change.